plugin/src/test/java/com/h3xstream/findsecbugs/xss/XssServletDetectorTest.java

```diff
@@ -19,255 +19,250 @@
 
 import com.h3xstream.findbugs.test.BaseDetectorTest;
 import com.h3xstream.findbugs.test.EasyBugReporter;
-import java.util.Arrays;
-
 import com.h3xstream.findsecbugs.FindSecBugsGlobalConfig;
+import java.util.Arrays;
 import org.testng.annotations.AfterMethod;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
-import static org.mockito.Mockito.*;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
 
 public class XssServletDetectorTest extends BaseDetectorTest {
 
-    @BeforeMethod
```

Comment:
@bradflood Everything looks good. If you could format the source to use 4 spaces, this would make the addition more clear.